docs/configuration/settings.rst

```diff
@@ -0,0 +1,586 @@
+
+Logging
+=======
+
++---------------------------------------+--------------------------------------------------------------------------+
+| Setting name                          | What does it do?                                                         |
++=======================================+==========================================================================+
+| cliquet.logging_renderer              | The Python *dotted* location of the renderer class that should be used   |
+| ``cliquet.logs.ClassicLogRenderer``   | to render the logs to the standard output.                               |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.newrelic_config ``None``      | `None`                                                                   |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.newrelic_env ``dev``          | `dev`                                                                    |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.statsd_prefix ``cliquet``     | The prefix to use when sending data to statsd.                           |
++---------------------------------------+--------------------------------------------------------------------------+
+| cliquet.statsd_url: ``None``          | The URL to use to connect to the statsd host. e.g.                       |
+|                                       | ``udp://localhost:8125``                                                 |
++---------------------------------------+--------------------------------------------------------------------------+
+
+Logging with Heka
+:::::::::::::::::
+
+Mozilla Services standard logging format can be enabled using:
+
+.. code-block:: ini
+
+    cliquet.logging_renderer = cliquet.logs.MozillaHekaRenderer
+
+
+With the following configuration, all logs are redirected to standard output
+(See `12factor app <http://12factor.net/logs>`_):
+
+.. code-block:: ini
+
+    [loggers]
+    keys = root
+
+    [handlers]
+    keys = console
+
+    [formatters]
+    keys = heka
+
+    [logger_root]
+    level = INFO
+    handlers = console
+    formatter = heka
+
+    [handler_console]
+    class = StreamHandler
+    args = (sys.stdout,)
+    level = NOTSET
+
+    [formatter_heka]
+    format = %(message)s
+
+
+Handling exceptions with Sentry
+:::::::::::::::::::::::::::::::
+
+Requires the ``raven`` package, or *Cliquet* installed with
+``pip install cliquet[monitoring]``.
+
+Sentry logging can be enabled, `as explained in official documentation
+<http://raven.readthedocs.org/en/latest/integrations/pyramid.html#logger-setup>`_.
+
+.. note::
+
+    The application sends an *INFO* message on startup, mainly for setup check.
```

Comment:
*Either* remove the comma and use parentheses:
> [...] startup (mainly for setup check).

*or* use words:
> [...] startup, which is useful chiefly as a setup check.

Your choice. :smile: